tests/conftest.py

```diff
@@ -481,3 +482,30 @@ def teardown_tftp_storage():
         assert teardown.values()[0]["rc"] == 0
 
     request.addfinalizer(teardown_tftp_storage)
+
+
+@pytest.fixture(scope="function")
+def setup_yum_content(request, ansible_module):
+    """ Setup/Teardown cusom yum repo for test_positive_content_migrate."""
+    prod_name = "Animals"
+    repo_name = "Animal_repo"
+    repo_url = FAKE_YUM0_REPO
+
+    ansible_module.command(f"hammer product create --organization-id 1 --name {prod_name}")
+    ansible_module.command(
+        f"hammer repository create --organization-id 1 --name {repo_name} --product {prod_name} "
+        f"--content-type 'yum' --url {repo_url} --download-policy 'immediate'"
+    )
+    setup = ansible_module.command(
+        f"hammer repository synchronize --organization-id 1 --product {prod_name} "
+        f"--name {repo_name}"
+    )
+    assert setup.values()[0]["rc"] == 0
+
+    def teardown_yum_content():
+        teardown = ansible_module.command(
+            f"hammer product delete --organization-id 1 --name {prod_name}"
+        )
```

Comment:
```suggestion
        teardown = ansible_module.command(f"hammer product delete --organization-id 1 --name {prod_name}")

```

Reply:
Pre commit is failing with this change https://github.com/SatelliteQE/testfm/pull/235/checks?check_run_id=2725568579

Reply:
I've rolled back to the original version.

Reply:
my bad ;)